manual/src/cmds/runtime-tracing.etex

```diff
@@ -51,19 +55,20 @@ a particular query or operation takes longer than expected to complete.
 
 \section{s:runtime-tracing-architecture}{Architecture}
 
-The runtime tracing system conceptually consists of two parts: 1) the probes
-which emit events and 2) the events transport that ingests and transports
-these events.
+The runtime tracing system conceptually consists of two parts: 1) the event
+sources which emit events and 2) the events transport that ingests and
```

Comment:
Nit: "events transport" reads weirdly to me, I would write "event transport", but then I'm not a native writer. (Or maybe just "transport" ?)

Reply:
FWIW, this isn't particularly bothering me either way

Reply:
Thanks! Let's leave it as-is then to minimize work.